### DISCUSSION TITLE
"OutputDirectory" parameter would like to be used even in dotnet-test-integration mode.

### DISCUSSION THREAD

**Discussion by norainu98** [SteveGilham/altcover#199] [Ideas]
Currently, "OutputDirectory" is below in dotnet-test-integration mode at v8.6.95.

- inplace=false:`$([System.IO.Path]::Combine($(TargetDir), '__Instrumented'))_$(ProjectName)`
- inplace=true: `$([System.IO.Path]::Combine($(TargetDir), '__Saved'))_$(ProjectName)`

Unfortunately, Since my some projects name is long, the path exceeds 256 characters in my environment.
So, I would like to use **[/o[utputDirectory]=VALUE]** such as the command line.

I would good if it could be defined as **p:AltCoverOutputDirectory=PATH**.

I try wrote a patch for build/AltCover.targets.
[AltCover.targets.patch](https://github.com/SteveGilham/altcover/files/13562574/AltCover.targets.patch)
For PR, Since I don't know this cominuty's rule for PR. 
If I also can push PR and it's best, I will push it.

Please consider above.
Regards.

BTW, sometimes removal of AltCover.Recorder.g.dll fails.
I don't found a reason, it seems that it is being used by another process.

**Comment by SteveGilham** [comment]
After emerging from winter hibernation, I've looked over this suggestion, and implemented a change to the same effect, permitting a substitution for the `$(TargetDir)` location - the subdirectory names still being provided automatically, so allow for solution builds to a single output location.  The change also extends the scripting APIs so that the location can be specified in more typesafe composition of the command line.

This is commit [0536248d5146d649a9b5247cbf1e29e0834ada6e](https://github.com/SteveGilham/altcover/commit/0536248d5146d649a9b5247cbf1e29e0834ada6e) and will be present in the next release.

**Reply by SteveGilham** [reply]
Now released as[ v8.7.3](https://www.nuget.org/packages/altcover/8.7.3#releasenotes-body-tab).
